src/middleware/upload.ts

```diff
@@ -0,0 +1,3 @@
+import multer from "multer";
+
+export const upload = multer({ dest: 'uploads/' });
```

Comment:
what did you make this file for? we already have uploadPhotos.js file in middleware folder. you need to restructure your code, so cloudinary stuff has its own space